geometry.h

```diff
@@ -23,6 +23,7 @@ class Geometry
         size_t                                                  _numVertex;
         size_t                                                  _numIndex;
         bool                                                    _useIndex;
+        bool													_useTexCoord;
```

Comment:
You don,t need this variable. Use VertexArray::hasAttribute(Attribute::ATTRIB_TEX ) instead